hw/ip/pinmux/doc/checklist.md

```diff
@@ -76,10 +76,10 @@ Security      | [SEC_RND_CNST][]        | N/A         |
 
  Type         | Item                         | Resolution  | Note/Collaterals
 --------------|------------------------------|-------------|------------------
-Security      | [SEC_CM_ASSETS_LISTED][]     | Not Started |
-Security      | [SEC_CM_IMPLEMENTED][]       | Not Started |
-Security      | [SEC_CM_RTL_REVIEWED][]      | Not Started |
-Security      | [SEC_CM_COUNCIL_REVIEWED][]  | Not Started |
+Security      | [SEC_CM_ASSETS_LISTED][]     | Done        |
```

Comment:
i wonder if we need to make an exception for blocks like pinmux / sysrst_ctrl.
These aren't security blocks in the typical sense, but they do technically serve specific system level security functions (even if not hardened ones).

Let me think about these a bit and get back to you.

Reply:
My gut feeling is that these don't count for D2S.